packages/runtime-common/query.ts

```diff
@@ -307,9 +307,9 @@ function assertEveryFilter(
       `${pointer.join('/') || '/'}: every must be an array of Filters`,
     );
   } else {
-    filter.every.every((value: any, index: number) =>
-      assertFilter(value, pointer.concat(`[${index}]`)),
-    );
```

Comment:
`.every` is problematic because it expects to return a boolean. So, not all assertions are being run thru